src/css/Navbar.css

```
@@ -107,52 +40,61 @@
   pointer-events: none;
 }
 
-@media only screen and (max-width: 800px) {
-
-  #navbar-links li::before,
-  #navbar-links li::after {
-    width: 25px;
-  }
+.nav-item:before {
+  transition: .3s ease;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  background: var(--offwhite);
+  content: "";
+  opacity: .1;
 }
 
-.navbar *,
-.navbar *:after,
-.navbar *::before {
-  -webkit-box-sizing: border-box;
-  -moz-box-sizing: border-box;
-  box-sizing: border-box;
+.nav-item:after {
+  transition: .3s ease;
+  position: absolute;
+  bottom: 0;
+  top: 0;
+  right: 0;
+  background: var(--offwhite);
+  content: "";
+  opacity: .1;
 }
 
-#navbar-links li:hover::before,
-#navbar-links li:hover::after,
-#navbar-links li:focus::before,
-#navbar-links li:focus::after {
+.nav-item:hover::before {
+  left: 50%;
   opacity: 1;
+  height: 100%;
+  transform: rotate(90deg) translateY(50%);
+  transform-origin: 0 100%;
+}
+.nav-item:hover::after {
+  /* right: 50%; */
+  height: 100%;
+  opacity: 1;
+  right: 50%;
+  transform: rotate(-90deg) translateY(-50%);
+  transform-origin: 0 0;
 }
 
-#navbar-links li:hover::before,
-#navbar-links li:focus::before {
-  left: 50%;
-  -webkit-transform: rotate(0deg) translateX(-50%);
-  -moz-transform: rotate(0deg) translateX(-50%);
-  transform: rotate(0deg) translateX(-50%);
+#main-logo {
+  height: 3em;
+  padding-left: 10px;
+  padding-top: 5px;
 }
 
-#navbar-links li:hover::after,
-#navbar-links li:focus::after {
-  right: 50%;
-  -webkit-transform: rotate(0deg) translateX(50%);
-  -moz-transform: rotate(0deg) translateX(50%);
-  transform: rotate(0deg) translateX(50%);
+@media only screen and (max-width: 600px) {
+  .navbar img {
+    height: 2em;
+  }
 }
 
-#navbar-links li a.active {
-  /* TODO: Make this work so that it looks like when it's hovered when on that page */
-  /* color: var(--red); */
-  right: 50%;
-  -webkit-transform: rotate(0deg) translateX(50%);
-  -moz-transform: rotate(0deg) translateX(50%);
-  transform: rotate(0deg) translateX(50%);
+@media only screen and (max-width: 450px) {
+  .navbar img {
```

You renamed stuff, so this no longer actually points to anything and needs to be renamed as such.

```suggestion
  #main-logo {
```